README.md

```diff
@@ -8,11 +8,30 @@ Notably, the multi-sig process on Flow is associated with two principal accounts
 
 At present, representatives from Blocto, Ichi, FindLabs, Flow Foundation, and an individual member (`@bluesign`) collectively hold multi-sig authority over the service account. Each signer is allocated a weight of 250 units, and to validate a transaction, a cumulative total of 1000 units is necessary. Therefore, a minimum of four signers must jointly provide their signatures to authorize the transaction. The transactions deliberated by these signatories may be related to setting transaction fees, managing/ slashing rewards, updating network-level contracts, adding new nodes, driving height coordinated and other node software upgrades, etc; see [account page](https://developers.flow.com/build/basics/accounts#service-accounts) for more details on the powers and abilities of the Flow service account.
 
-**Role of a multi-signer**
+## Role of a multi-signer
 
 Multi-signers on Flow have a direct hand in shaping Flow’s future. They represent the community, oversee protocol changes, and ensure decisions align with the interests of all Flow participants. They regularly join meetings, some scheduled and others on short notice. On average and under normal circumstances, signers are expected to attend 1-2 meetings per month, each lasting about 10-15 minutes. During busier periods, such as major protocol updates, there may be more frequent and short-notice meetings.
 
-**How to nominate?**
+## 🛠️ Service Committee
+The group of multi-signers are effectively referred to as the service committee.
+
+## 📋 Policy for Multi-signers
+An efficient and smooth multi-sign process is essential to ensure upgrades on Flow are rolled out in a timely manner. Reviewing, approving, and signing transactions requires a time commitment from each member of the service committee.
+
+Equally important is preserving the decentralized nature of the multi-sign process. The service committee must reflect the diversity and independence of the Flow ecosystem and avoid any form of centralization of signing power.
+
+To that end, the following policy applies:
+
+1. Meeting Participation
+Each signer is expected to not miss more than 10 consecutive multi-signing meetings.
+
+2. Inactive Signer Removal
+If a signer exceeds this threshold, the rest of the committee may propose their removal through a governance FLIP.
```

Comment:
```suggestion
If a signer exceeds this threshold, the rest of the committee may propose their removal through a governance FLIP. The inactive key will be notified and have a grace period of two weeks to resolve any issue that may prevent them from signing or to work with the signing committee to find their replacement if they no longer wish to sign. In the event that a key holder cannot be reached, the signing committee will nominate a replacement signer. The replacement signer should ideally represent the same area of interest that the previous signer did (for example, a signer that works on a wallet should be replaced with another wallet when possible). 
```

Reply:
> 

I have update the second clause to include this line `The inactive key will be notified and have a grace period of two weeks to resolve any issue that may prevent them from signing`, however, I haven't included the clause of finding a replacement and that I think is separate and covered by the `How to nominate` section.